packages/cli/test/js-format.config.mock.cjs

```diff
@@ -0,0 +1,51 @@
+/**
+ * This config file is here to demonstrate the CommonJS version of the 4 different supported versions ('ts' | 'mjs' | 'cjs' | 'js')
+ *
+ * Usage:
+ * npx ./dist/packages/cli collect --config=./packages/cli/test/js-format.config.mock.cjs
+ */
+
+const outputDir = 'tmp';
+module.exports = {
+  upload: {
+    organization: 'code-pushup',
+    project: 'cli-cjs',
+    apiKey: 'dummy-api-key',
+    server: 'https://example.com/api',
+  },
+  persist: { outputDir },
+  plugins: [
+    {
+      audits: [
+        {
+          slug: 'command-object-audit-slug',
+          title: 'audit title',
+          description: 'audit description',
+          docsUrl: 'http://www.my-docs.dev',
+        },
+      ],
+      runner: {
+        command: 'node',
+        args: [
+          '-e',
+          `require('fs').writeFileSync('${outputDir}/out.json', '${JSON.stringify(
+            [
+              {
+                title: 'dummy-title',
+                slug: 'command-object-audit-slug',
+                value: 0,
+                score: 0,
+              },
+            ],
+          )}')`,
+        ],
+        outputFile: `${outputDir}/out.json`,
```

Comment:
Please use `echo` instead of `node -e` and `path.join` instead of hard-coded `/` separator. See changes in #131.

(Same goes for all these `js-format.config.mock.*` files.)